bootstrap/internal/controllers/rke2config_controller.go

```diff
@@ -413,7 +414,18 @@ func (r *RKE2ConfigReconciler) handleClusterNotInitialized(ctx context.Context,
 		Certificates: certificates,
 	}
 
-	cloudInitData, err := cloudinit.NewInitControlPlane(cpinput)
+	var cloudInitData []byte
```

Comment:
Should we not change the variable name from `cloudInitData` to `userData` to avoid confusion?